src/qt/rpcconsole.cpp

```diff
@@ -523,6 +525,16 @@ void RPCConsole::setNumBlocks(int count, const QDateTime& blockDate)
     ui->lastBlockTime->setText(blockDate.toString());
 }
 
+void RPCConsole::setMempoolSize(int numberOfTxs, size_t dynUsage)
+{
+    ui->mempoolNumberTxs->setText(QString::number(numberOfTxs));
+
+    if (dynUsage < 1024*1024)
+        ui->mempoolSize->setText(QString::number(dynUsage/1024.0, 'f', 2) + " KB");
+    else
+        ui->mempoolSize->setText(QString::number(dynUsage/1024.0/1024.0, 'f', 2) + " MB");
```

Comment:
But it's used less commonly, so MB is fine as well. Just be consistent, which is what the NIT is complaining about. You can't parse the command line in MB (1e6) and then display MB (10^20)